src/main/java/Eliz/Eliz.java

```diff
@@ -0,0 +1,331 @@
+            break;
+        case "deadline":
+            t = createDeadline(taskType, splitTwoSections[1]);
+            break;
+        case "event":
+            t = createEvent(taskType, splitTwoSections[1]);
+            break;
+        default:
+            return null;
+        }
+        return t;
+    }
+
+    public static Task createTodo(String taskType, String line) {
+        Todo newTodo = new Todo(line);
+        return newTodo;
+    }
+
+    public static Task createDeadline(String taskType, String line) {
+        String newDescription = line;
+        if (line.contains("/")) {
+            String[] splitTwoSections = line.split("/", 2);
+            newDescription = splitTwoSections[0] + "(" + splitTwoSections[1] + ")";
+        }
+        Deadline newDeadline = new Deadline(newDescription);
+        return newDeadline;
+    }
+
+    public static Task createEvent(String taskType, String line) {
+        String newDescription = line;
+        if (line.contains("/")) {
+            String[] splitTwoSections = line.split("/", 2);
+            newDescription = splitTwoSections[0] + "(" + splitTwoSections[1] + ")";
+        }
+        Event newEvent = new Event(newDescription);
+        return newEvent;
+    }
+
+    public static void botIntroduction() {
+        String logo = " ____    __       __     ______ \n"
+                + "|  __|  |  |     |  |   |___  /\n"
+                + "| |__   |  |     |  |      / /  \n"
+                + "| |__|  |  |     |  |     / /  \n"
+                + "| |__   |  |__   |  |    / /___\n"
+                + "|____|  |_____|  |__|   |______|\n";
+
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Eliz");
+        System.out.println("I am your personal task list recorder! If you need help, type [/help]");
+        System.out.println("What task would you like me to note down for you?");
+    }
+
+    public static void markOrUnmark(String line, ArrayList<Task> tasks) {
+        if (line.contains("unmark")) {
+            System.out.println("OK, I've marked this task as not done yet:");
+            unmarkATask(line, tasks);
+        } else {
+            System.out.println("Nice! I've marked this task as done:");
+            markATask(line, tasks);
+        }
+    }
+
+    public static void displayCommands() {
+        System.out.println("Commands to add tasks: ");
+        System.out.println("todo [TASK_NAME]");
+        System.out.println("event [TASK_NAME]");
+        System.out.println("deadline [TASK_NAME]/[DATE]");
+        System.out.println(" ");
+        System.out.println("Command to display list: ");
+        System.out.println("list");
+        System.out.println(" ");
+        System.out.println("Commands for todo tasks: ");
+        System.out.println("mark [TASK_LIST_NUMBER]");
+        System.out.println("unmark [TASK_LIST_NUMBER]");
+    }
+
+    public static void delete(String line, ArrayList<Task> tasks) {
```

Comment:
It would help to add [javadoc comments](https://se-education.org/guides/conventions/java/basic.html#comments) here to explain what is deleted.